user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,129 @@
+
+## Firewall
+
+Are all machines online? Can they talk to each other? Please check this beforehand. If you're using a firewall, please make sure that these ports are open and the machines accept traffic on these:
+
+__Worker:__
+
+- Port 22 for SSH access
+
+__Platform machine__:
+
+- Port 22 for SSH access
+- Port 443 for HTTPS traffic
+- Port 80 for HTTP traffic
+- Port 4567 & 5672 for RabbitMQ traffic
+- Port 8800 for management dashboard https traffic
+
+## Hardware
+
+Our platform machine has these hardware requirements:
+
+- Minimum 16GB of RAM
+- 8 CPU cores
+- Min. 40GB of available disk space
+
+When you run `top` on the platform machine, please check if enough RAM is available and that the system load is not too high.
+Did you notice an increased system load recently?
+
+## Docker
+
+We use a tool called [Replicated](https://www.replicated.com/) to distribute Travis CI Enterprise. It also provides the management dashboard you're using to configure certain things.
+
+Replicated and Travis CI Enterprise are both shipped as Docker containers. Therefore we have to rely on a perfectly working Docker installation. In the past we've seen incidents where the customers production system became unavailable because Docker related configuration has changed for various reasons.
+
+Therefore we need to have some things in line:
+
+- Your currently used Docker version needs to be at least 1.7 on both platform and worker machines.
+- For the platform machine, these port bindings have to be in place (run `sudo docker ps -a` and look for a container whose name starts with `quay.io-travisci-te-main`):
+	- 0.0.0.0:80->80/tcp
+	- 0.0.0.0:443->443/tcp
+	- 0.0.0.0:4567->4567/tcp
+	- 0.0.0.0:5672->5672/tcp
+	- 0.0.0.0:32770->22/tcp
+- If the above port bindings are not there, please let us know. Then we'll figure out how to get them back.
+- An important topic is the used storage driver. In the past we've heavily relied on the `devicemapper` storage driver. At some point Docker changed their default storage driver to `overlay`. However this only works well if your Linux kernel is at least version 3.18. Below it *can* work but there's no definite guarantee. To find out the used storage driver, please run `sudo docker info | grep "Storage Driver"`. If your current driver is `overlay`, please check your Linux kernel version via `uname -a`.
+
+## Worker
+The `travis-worker` binary on the worker machine(s) must be up to date.
+Please run `sudo apt-get install --only-upgrade travis-worker`. This will update it to the latest version.
+
+__Docker__:
+
+Since the worker machine runs all your builds in Docker containers, the Docker version must be up to date and properly configured. We also need to have the `devicemapper` or `overlay` storage driver configured here, together with Linux kernel 3.18 or later.
```

Comment:
Like the platform machines, `aufs` is our default storage driver still. Though there are some `devicemapper` installs out there from our installer. I think only verizon is using Overlay with their customer worker setup.